### ISSUE TITLE
How to manage the "back" from the browser?

### ISSUE THREAD

**Issue by geomagilles** [kwebio/kweb-core#214]
Hi. on my Kweb app, when I use the back arrow, the url is updated but not the content. I'm not sure at all how to handle this event, can you help please?

**Comment by sanity** [comment]
Duplicate of #30, it's been a known issue for quite a while, it shouldn't be a difficult fix but it hasn't been a squeaky wheel, it does need to be fixed before 1.0 though which we're approaching.

@geomagilles , how urgently do you need a fix?

**Comment by geomagilles** [comment]
@sanity thx for your quick response. It's needed but not urgent. Any timeframe for 1.0?

**Comment by sanity** [comment]
No formal deadline but I think we're probably weeks away, @Derek52 is currently working on #151 which is the last big blocker for it. 

I might be able to tackle this myself depending on other obligations, or @Derek52 might be able to address it after he's done with #151.

**Comment by geomagilles** [comment]
Regarding the blocking issues, I have display bugs with 
`render(data) { }` when data is updated quickly by parallel coroutines. I guess it's the same than #202

**Comment by sanity** [comment]
Yes, IIRC this should be fixed by #151 also.

**Comment by geomagilles** [comment]
@sanity FYI, I'm currently writing a SPA using your framework and tailwindui. it works great congrats!!
The main issue is that I have to re-invent the way to write such app. How to organize the menu, the panels, the states, and so on. If you want, this could serve as an example

**Comment by sanity** [comment]
That would be really great, we definitely need to establish best practices. Will your SPA be open source?

**Comment by geomagilles** [comment]
I use a [Common Clause licence](https://commonsclause.com) - it's similar to open source but restrict cloud providers to use it

**Comment by geomagilles** [comment]
I would be happy to have your reviews on implemented practices btw

**Comment by sanity** [comment]
Ah, interesting - I'd heard of AGPL but not this. That sounds fine, would be happy to link to your project as an example, will just need to highlight the license differences to avoid confusion.

What's your timeline? Is the src currently available?

**Comment by geomagilles** [comment]
it's a dashboard for infinitic.io  - https://github.com/infiniticio/infinitic (on a new module Infinitic-dashboard not pushed yet). I hope to finalize it this summer.

**Comment by sanity** [comment]
Very cool. I dabbled in a vaguely related concept a few years back - http://swarmframework.org/.

If you don't mind me asking, is this a personal project, or work related?

**Comment by geomagilles** [comment]
It's my new project after having stopped zenaton.com. Currently, I'm doing paid services around the framework capabilities, but my intentions are to create soon a venture around it. It's similar to temporal.io - except that I use Apache Pulsar as the underlying platform. Doing so I'm benefit of the quick development of Pulsar and can focus on the workflow engine. I've looked at your video, it's exciting you were working on similar topics a few years back. I'd be happy to do a call if you are still interested

**Comment by sanity** [comment]
Definitely, could you ping me via ian@uprizer.com, or let me know your email?

**Comment by sanity** [comment]
Appears to work now in `master`, still some exceptions to debug and unit tests need to be completed before release.
